Build error in Fedora (stdgpu)

**Describe the bug**
When building with `BUILD_CUDA_MODULE=ON` on Fedora 35, a build error occurs.
(Build error does not occur on Ubuntu)
```
make[2]: *** No rule to make target 'stdgpu/lib64/libstdgpu.a', needed by 'bin/ConvertPointCloud'.  Stop.
make[1]: *** [CMakeFiles/Makefile2:3308: cpp/tools/CMakeFiles/ConvertPointCloud.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

**To Reproduce**
```
git clone https://github.com/isl-org/Open3D.git
mkdir build && cd build
cmake -DBUILD_CUDA_MODULE=ON ..
make
```

**Expected behavior**
Build succeeds with `BUILD_CUDA_MODULE=ON`.

**Screenshots**

**Environment (please complete the following information):**

- Operating system: Fedora 35
- Python version: Python 3.9
- Open3D version: f4459dfcf1536999641fe0234a67dee865f4de3d
- Is this remote workstation?: no
- How did you install Open3D?: build from source
- Compiler version (if built from source): gcc version 11.2.1 20210728 (Red Hat 11.2.1-1) (GCC)

**Additional context**
In [stdgpu.cmake](https://github.com/isl-org/Open3D/blob/afee932e7582e457408bfd5e4e5420bdf5d6b93f/3rdparty/stdgpu/stdgpu.cmake), specify `${Open3D_INSTALL_LIB_DIR}` for `STDGPU_LIB_DIR`.
https://github.com/isl-org/Open3D/blob/afee932e7582e457408bfd5e4e5420bdf5d6b93f/3rdparty/stdgpu/stdgpu.cmake#L31

In Fedora, `${Open3D_INSTALL_LIB_DIR}` is `lib64` (On Ubuntu it is `lib'.).
However, for stdgpu, `lib` is specified in the path.
https://github.com/stotko/stdgpu/blob/e10f6f3ccc9902d693af4380c3bcd188ec34a2e6/CMakeLists.txt#L139
This build error occurs due to the difference in the path.  
I would like to send a pull request.

--- a/3rdparty/stdgpu/stdgpu.cmake
+++ b/3rdparty/stdgpu/stdgpu.cmake
@@ -28,5 +28,5 @@ ExternalProject_Add(
 
 ExternalProject_Get_Property(ext_stdgpu INSTALL_DIR)
 set(STDGPU_INCLUDE_DIRS ${INSTALL_DIR}/include/) # "/" is critical.
-set(STDGPU_LIB_DIR ${INSTALL_DIR}/${Open3D_INSTALL_LIB_DIR})
+set(STDGPU_LIB_DIR ${INSTALL_DIR}/lib)
 set(STDGPU_LIBRARIES stdgpu)

